open-sphere-base/analysis/src/main/java/io/opensphere/analysis/baseball/BaseballPanel.java

```diff
@@ -128,11 +135,47 @@ private Node createDataElementList()
      */
     private Node createDataArea()
     {
-        myDataView.setCellFactory((param) ->
+        TableColumn<BaseballDataRow, String> fieldColumn = new TableColumn<>("Field");
+        fieldColumn.setCellValueFactory(data -> data.getValue().fieldProperty());
+        myDataView.getColumns().add(fieldColumn);
+
+        TableColumn<BaseballDataRow, String> valueColumn = new TableColumn<>("Value");
+        valueColumn.setCellValueFactory(data -> data.getValue().valueProperty());
+        myDataView.getColumns().add(valueColumn);
+
+        myDataView.setColumnResizePolicy(TableView.CONSTRAINED_RESIZE_POLICY);
+        myDataView.getSelectionModel().setCellSelectionEnabled(true);
+
+        myDataView.setOnMouseClicked(event ->
         {
-            BaseballDataRow row = new BaseballDataRow();
-            return row;
+            if (event.getButton().equals(MouseButton.PRIMARY) && event.getClickCount() == 2)
```

Comment:
If you just Ctrl+C the text, does it get copied?  If not, should we implement it so Ctrl+C does the copy instead of double click?  That would be more Windows over Linuxy.  Just a thought.

Reply:
The text in the new table is actually not selectable, similar to how it is in the analyze tool. It used to be possible when it was text fields, but the new table doesn't use them. Unless you're suggesting that Ctrl+C copies the cell block without needing to highlight the test itself, which would probably work.

Reply:
If it's not selectable, then it's gonna suck either way.  I guess double click is good enough for now but I'm thinking we should have a follow on ticket to make it more intuitive.

Reply:
Added Ctrl+C for copying the selected cell to clipboard. Text is still unselectable itself, but that key combination will copy the cell. No message popup happens copy is done using Ctrl+C. I left in the double click due to the similarity between this and the analyze tool which does the same thing, but should that removed?

Reply:
Probly fine to leave it in.  Personally the double click copy in analyze tool seems weird to me, but whatevs.